### DISCUSSION TITLE
Welcome to Java-Assignment Discussions!

### DISCUSSION THREAD

**Discussion by kafle1** [internsathi/Java-Assignment#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tamanghawana09** [comment]
### Hello fellow developers and enthusiasts,
I am Hawana Tamang, currently studying BIT (4th semester) at Kist College.
And, I'm willing to collaborate and contribute to open source projects and moreover learn and upskill the knowledge

**Comment by ujjwol75** [comment]
HI everyone, Its Ujjwol Neupane. I am currently studing BSC CSIT 8th semester at Samriddhi College, Bhaktapur. I feel honour to be part of this community.
